docs/api/ReactWrapper/text.md

```diff
@@ -22,7 +22,7 @@ expect(wrapper.text()).to.equal('important');
 
 ```jsx
 const wrapper = mount(<div><Foo /><b>important</b></div>);
-expect(wrapper.text()).to.equal('<Foo />important');
+expect(wrapper.text()).to.equal('important');
```

Comment:
Good catch! The text would actually also include whatever text `Foo` rendered; perhaps this should also include a definition for a `Foo`?